src/providers/infoparser.ts

```diff
@@ -117,7 +117,9 @@ export class REPLInfoParser {
       } else {
         hover.appendText('\n');
       }
-      hover.appendMarkdown(this._docString);
+      if (this._docString) {
+        hover.appendMarkdown(this._docString);
+      }
```

Comment:
I'd like for us to show _(No doc string)_ here. I heard you about that clojure-lsp does not do that and that other language extension doesn't either. But I think it is good for the user to see it stated, so that you don't start wondering about if Calva is just not showing them. And also a good reminder to add a doc string.